docs/howto/release-git-lfs.md

```diff
@@ -337,3 +337,269 @@ and then use that branch as the base for the PATCH release.
+
+  2. Create a temporary private fork of this repository from the draft
+     security advisory page, and use the fork to develop a resolution
+     of the vulnerability.
+
+  3. Create two PRs in the private fork, one to remediate the vulnerability
+     in the `main` development branch of the public `git-lfs/git-lfs`
+     repository, and one to do so in the appropriate `release-M.N` branch.
+
+     The PR which targets the release branch should have a final, extra
+     commit which adds an entry for the new PATCH release version in the
+     `CHANGELOG.md` file, describing the release and the security fix it
+     contains.  This commit should also include the changes generated
+     by the `script/update-version vM.N.P` command, as per the second step
+     in our standard release process.
+
+     If the `release-M.N` branch does not exist in the public repository,
+     create it as per our regular PATCH release process, and then create
+     the second PR in the private fork against the new branch.  Be prepared
+     to proceed relatively quickly, since the appearance of the new branch
+     serves as a public notice that we may be publishing a PATCH release
+     shortly.
+
+  4. From the draft security advisory page, use the "Merge pull requests"
+     option to merge both PRs simultaneously.
+
+     Note that both the release branch and the `main` branch in the private
+     fork must be up-to-date with the corresponding branches in the public
+     repository in order for the GitHub UI to determine whether the PRs have
+     no merge conflicts.
+
+  5. Publish the security advisory.
+
+  6. Follow our standard release process, starting with the step in which
+     we create a GPG-signed tag named `vM.N.P` on the merge commit to the
+     `release-M.N` branch.
+
+  7. After publication of the new release, update the `git-lfs.com` home
+     page with a banner message regarding the security PATCH release
+     by pushing a change to the `_includes/home/secondary.html` file
+     in the `git-lfs/git-lfs.github.com` repository.
+
+  8. Create a new discussion in the "Announcements" section of the GitHub
+     discussion [forum](https://github.com/git-lfs/git-lfs/discussions)
+     describing the security PATCH release, and pin the discussion for
+     all categories in the forum.
+
+### Building security PATCH versions under embargo
+
+When coordinating the release of a security patch with one or more
+other projects, we must not follow the processes described above as
+they depend on the GitHub Actions release workflow in our public
+`git-lfs/git-lfs` repository.  Instead, we use a private repository
+with a modified release workflow to build the binaries and packages
+for our new version so that we can share them with the other projects
+in advance of the coordinated release date.
+
+  1. Open a draft security advisory, apply for a CVE identifier, and
+     create a private fork in which to develop a resolution of the
```

Comment:
Is "which to develop" correct? It reads a bit odd as non-native speaker.

Reply:
I'm pretty sure it's correct English grammar, although perhaps a bit formal or abstruse.

I'm no expert, but from a bit of research, I believe the "in which" is a prepositional phrase since it begins with the preposition "in" and ends with the object "which", where the "which" introduces a relative clause, in this case, the full infinitive "to develop".  (Corrections to that description are welcome!)

I've rephrased the sentence, though, as three individual sentences, which I hope makes it easier to read for everyone.